src/providers/aptos.ts

```diff
@@ -98,10 +105,14 @@ export default class AptosProvider extends BloctoProvider implements AptosProvid
     this.address = undefined;
   }
 
-  async signAndSubmitTransaction(transaction: any): Promise<{ hash: HexEncodedBytes }> {
+  async signAndSubmitTransaction(
+    transaction: any,
+    { max_gas_amount }:TxOptions = { max_gas_amount: '0' }
+  ): Promise<{ hash: HexEncodedBytes }> {
```

Comment:
you can pass all the options to server, no need to extract `max_gas_amount`

Reply:
according to the examle [here](https://github.com/hippospace/aptos-wallet-adapter/blob/cdf603d98682354bf4a5ef74303701c8636c1d2e/packages/wallet-nextjs/pages/index.tsx#L89-L91) , it seems that there'll be two properties in txOption, but I'm not sure there'll be other properties, so I also added ` [key: string]: any` in interface, it's that okay?  ( see the commit [here](https://github.com/portto/blocto-sdk/pull/38/commits/a7dcd00328f61c0b385d6cd8871f80b48c873b0d))